package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "version": "1.0.0",
+  "version": "1.1.0",
```

Comment:
Hmm, if this is a semver breaking release shouldn't this be `2.0`?

Reply:
Technically, yes. But since its such a small breaking change that may be reverted in the near future I was thinking of having a note but keeping bumping minor like we do here.

Reply:
I'm of two minds on this. One voice in my head says "Last famous words! A change is either breaking or it's not, no such thing as a 'small' breaking change! Don't stray from semver." and the other is saying "Release 2.0 for this is just silly; and not releasing is also silly, so what do you want to do?".

As this is a fix for a regression downstream maybe we should treat this as a patch release and change the wording in the change log from `**BREAING**` to something less alarming sounding and instead spend some time explaining the circumstance(s) where this could break user code?

Reply:
I'm with David that this should be rather 2.0.0 or 1.0.1, pending on how we judge the breakiness:

+ If the breaking change is actually fixing a bug, rather than affecting a contract with end users, then it's 1.0.1.
+ If it's actually breaking, no matter how small, then it's 2.0.0.

I get that going from 1.0.0 to 2.0.0 quickly _feels_ weird, but I also don't think we necessarily guarantee 100% api correctness for end users, so they should expect breaking changes down the road. If we make it a practice now then switching from 19.0.0 to 20.0.0 later will feel less weird.

Reply:
Zeke, can you please describe exactly why you consider this change "breaking"? As Maciej says, think in terms of fixing unintended/"buggy" behavior versus breaking "valid" end-user expectations, even if they are not stated explicitly. We shouldn't support people relying on buggy behavior in production, but we should endeavor not to break existing workflows. If we believe this change will break valid, existing workflows, we should bump the major version.

Reply:
I guess the issue is that the intended behavior in the first place is not clear IMO. We are going from having metadata with pokadot-js specific type aliases to having the plain substrate metadata (at least from what I understand.) What I am not clear about is if our users expect pokadot-js specific metadata (or if anyone is even programmatically using this endpoint).
 
The other factor here is that we already released a v1.1.0 on NPM to quickly fix an issue with dependencies, and NPM does not allow you to release a decreased semver version, so 1.0.1 is out of the option there and I think would be confusing to have something else here

Reply:
> What I am not clear about is if our users expect pokadot-js specific metadata (or if anyone is even programmatically using this endpoint).

Sounds like it is a breaking change, just one that likely affects very few people. If so, I think we should go with 2.0 (and not be coy about using major numbers; after all integers are plentiful and cheap).

DQ: doesn't npm allow releasing patches to older versions? Like, relase v3.2.0 and also v2.9.1?

Reply:
We could release as a v2-rc or something as well

Reply:
>. DQ: doesn't npm allow releasing patches to older versions? Like, release v3.2.0 and also v2.9.1
I think this is correct